README.md

```diff
@@ -20,6 +20,12 @@ If LRN starts with:
 - 'D' - Stub will send an 'Accepted' and an 'Additional Documents Required' notifications
 - 'G' - Stub will send an 'Accepted' notification
 - 'Q' - Stub will send a 'Query Notification Message' notification
+- 'I' - Stub will send a 'Awaiting exit results' notification
```

Comment:
`Q` should be after `P`, isnt'it? :smile:

Reply:
Inserted in wrong place, good catch :+1: